nifi-registry/nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/api/BucketResource.java

```diff
@@ -92,6 +92,32 @@ public Response createBucket(
         return Response.status(Response.Status.OK).entity(createdBucket).build();
     }
 
+    @POST
+    @Path("migrate")
```

Comment:
Same as previous comment about using a query param on the existing create bucket and being consistent with calling it `preserveSourceProperties`